splunk/splunk_test.go

```diff
@@ -0,0 +1,79 @@
+package splunk
+
+import (
+	_ "embed"
+	"fmt"
+	"log"
+	"os/exec"
+	"regexp"
+	"testing"
+
+	"github.com/redhat-appstudio/segment-bridge.git/containerfixture"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+//go:embed splunk_container_template.yaml
+var splunkServiceManifest string
+var countPattern = regexp.MustCompile(`(?m)^count\s+(\d+)$`)
+
+const NotUpErrorMsg = "The %s instance is not up, cannot verify indexing for tests."
+
+func WithSplunkContainer(t *testing.T, test_func func(*testing.T, containerfixture.FixtureInfo)) {
+	containerfixture.WithServiceContainer(t, splunkServiceManifest, test_func)
```

Comment:
```suggestion
containerfixture.WithServiceContainer(
    t, splunkServiceManifest, func(t *testing.T, fi containerfixture.FixtureInfo) {
        endpoint := fmt.Sprintf("http://localhost:%s/%s", fi.ApiPort, ServiceStatusCheckPath)
        containerfixture.RequireServiceIsUp(t, endpoint, ServiceName, NotUpErrorMsg)
        test_func(t, fi)
    })
```

Also this kind of makes you see that you don't really need to be explicitly passing 't' into the test func in both fixtures because when defined as a closure it can always get `t` from the surrounding function that defines it.

Also small styling nitpicks - in Go you use `camelCase` and not snake_case` for variables. And you don't seem to be using consistent indentation, please make sure you use 4 spaces.

Reply:
Fixed, thanks for the explanation, it is more clear now! 
Obviously as seen by the rest of the code, I'm aware of the camelCase... but old habits die hard. will try to notice it more.